care/facility/api/serializers/patient.py

```diff
@@ -472,9 +473,20 @@ def save(self, **kwargs):
                 self.instance.save()
 
 
+class PatientNotesEditSerializer(serializers.ModelSerializer):
+    id = serializers.UUIDField(source="patient_note.external_id", read_only=True)
+    edited_by = UserBaseMinimumSerializer(read_only=True)
+
+    class Meta:
+        model = PatientNotesEdit
+        exclude = ("patient_note",)
+
+
 class PatientNotesSerializer(serializers.ModelSerializer):
+    id = serializers.CharField(source="external_id", read_only=True)
     facility = FacilityBasicInfoSerializer(read_only=True)
     created_by_object = UserBaseMinimumSerializer(source="created_by", read_only=True)
+    edits = PatientNotesEditSerializer(many=True, read_only=True)
```

Comment:
Check if we can make this into a separate API and call only when needed, no reason to query all the edits all the time. also need to look at N + 1 queries.